### DISCUSSION TITLE
Browser Tab Notifications

### DISCUSSION THREAD

**Discussion by KyleJSh** [deso-protocol/dips#134] [Q&A]
I notice that Twitter and YouTube have notifications in the browser tabs, showing if you have a message. Would love to see this on BitClout as well.

Ex:
(1) Home / Twitter

**Comment by KyleJSh** [comment]
![Screen Shot 2021-08-20 at 10 47 48 AM](https://user-images.githubusercontent.com/30233330/130277367-a5d7400f-21b0-4f9d-bf3b-e743e505ecf2.png)

It looks like this

**Comment by ItsAditya-xyz** [comment]
Would be a good addition but i think right now the logic to get the notification dot has been temporary turned off due to performance issues.
